src/component/parser/json/converter/GlobalTaskConverter.ts

```diff
@@ -14,21 +14,14 @@
  * limitations under the License.
  */
 import { TDefinitions } from '../../../../model/bpmn/json/BPMN20';
-import { ensureIsArray } from './ConverterUtil';
+import { ConvertedElements, ensureIsArray } from './utils';
 import { TGlobalTask } from '../../../../model/bpmn/json/baseElement/rootElement/globalTask';
 
-const globalTaskIds: string[] = [];
-
-export function isGlobalTask(id: string): boolean {
-  return globalTaskIds.includes(id);
-}
-
 export default class GlobalTaskConverter {
+  constructor(readonly convertedElements: ConvertedElements) {}
+
   deserialize(definitions: TDefinitions): void {
     try {
-      // Deletes everything in the array, which does hit other references. For better performance.
-      globalTaskIds.length = 0;
```

Comment:
We have to clean the maps before each new parsing, because it's keep the old converted elements of the previous file.
Some elements can be in conflict if there are the same id.

Reply:
Forgot my comment. I didn't see that we instantiate a new BPMNParser (and new ConvertedElements) at each loading ^^